packages/mdc-menu/simple/foundation.js

```diff
@@ -15,10 +15,34 @@
  * limitations under the License.
  */
 
+/**
```

Comment:
Just in general...`MDCSimpleMenuFoundation` is now 570 lines of code. Which is quite a lot. Is there any way to break out some piece of this into a new file? Open to ideas here...

Reply:
Might be feasible to genericize some of the broken-out auto-positioning functions to util functions?

Reply:
Separate PR, plz!

Reply:
Yeah, I'll create an issue, since I also realized that I think there are now some unused `util` functions anyway.